README.md

```diff
@@ -54,3 +54,25 @@ TEMPLATES = [
     }
 ]
 ```
+
+A custom folder to load the templates can be specified. This allows you to keep the URL acessible templates separately from template includes.
+
+``` python
+# As a subfolder in templates. Only searching inside `templates/pages/`
+TEMPLATE_FINDER_DIR = 'pages'
+```
+
+To specify an absolute directory outside templates, it must be added to the `DIRS` setting in `TEMPLATES`.
+``` python
+# Outside the templates folder
+TEMPLATE_FINDER_DIR = os.path.join(BASE_DIR, 'pages')
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [
+            os.path.join(BASE_DIR, 'templates'),
```

Comment:
Setting it as an absolute path is a safer choice and my preference.
Relative path works when you run the command in the correct directory but will fail when you run it from a different folder.

This command works with absolute, and fails to find template with relative:
`python developer.ubuntu.com/manage.py runserver`